uber/templates/jobs/staffers.html

```diff
@@ -70,7 +70,7 @@
 <div style="margin-left:10px">
     [D]: <i>Staffers marked with a "D" are department heads.</i> <br/>
     [M]: <i>Staffers marked with a "M" are assigned to multiple locations</i> <br/>
-    [T]: <i>Staffers marked with a "T" are trusted</i> <br/>
+    [T]: <i>Staffers marked with a "T" are trusted in at least one department</i> <br/>
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
 </div>
 
```

Comment:
Hmm, this is a summary report *for a specific department* though, so I imagine it would be better to show who's trusted here specifically.

We'll need to write a custom filter for that in ``uber/custom_tags.py``.